src/xdocs/config_naming.xml

```diff
@@ -903,6 +903,24 @@ class MyClass {
   final static int MYSELF = 100; // OK
   final static int myselfConstant = 1; // violation, name 'myselfConstant' must match pattern
                                        // '^log(ger)?$|^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+}
+        </source>
+        <p>
+         The following configuration is applicable by default.
+        </p>
+        <source>
+&lt;module name=&quot;ConstantName&quot;&gt;
+&lt;/module&gt;
+        </source>
+        <p>Code Example:</p>
```

Comment:
`Code Example:` -> Example:

Reply:
ok, I will do it.

Reply:
Also, shall I make another commit? or go to files changed and just edit it over there?

Reply:
replied below